### ISSUE TITLE
[BUG] DEX Slows down the ROM

### ISSUE THREAD

**Issue by byalpcngz** [ananjaser1211/hadesRom_Reports#72]
**Describe the feature**
Dex mode is too slow when the phone screen is on on my device or if a phone application is running on the screen

**Smartphone (please complete this feature following information part if you want device spesific):**
 - Device: Galaxy Note 8.

**Comment by AzzyC** [comment]
I also have a Galaxy Note 8, I was freely opening/using apps while DeX mode was on and using that too

**Comment by byalpcngz** [comment]
I have no problem using applications with dex mode turned on.  My problem is when the dex mode is turned on, the application is on my phone, if it is open, the mouse moves with a delay.

**Comment by ananjaser1211** [comment]
HDMI dex mode or DexOnPC ? and please attach logs when dex is running and you are trying to lunch apps

**Comment by twinnybloggers** [comment]
> I have no problem using applications with dex mode turned on. My problem is when the dex mode is turned on, the application is on my phone, if it is open, the mouse moves with a delay.

If u using dex mode, let phone to apps screen dont staying on homescreen may this will make your mouse feel delay . So firstly launch dex and change your phone home screen to app screen . 

Hope this usefull

**Comment by byalpcngz** [comment]
yes this method solves my problem but i liked working while watching video on the phone screen it would be nice to solve this problem

**Comment by corsicanu** [comment]
@byalpcngz .

**Comment by ananjaser1211** [comment]
DEX Seems to be locking the device down to 30 FPS.

As per telegram user @draeltv

"DeX which apparently leaves a big delay in touch and FPS in general. I use DeX with the cable and every time I turn it on at least on Hades Q, otherwise I would use Hades Q v2 normally, but my S8 is only used for games and I think it's cool to play on a screen / monitor. It already happened in Hades Q v1.0. And because of that, I went back to Hades P V12. Where it works normally. The problem is not the 4k monitor or anything else because I use a capture card to transmit an image leaving the device's GPU free of it and on Hades P it worked on all the screens I tested, but on hades Q it gives the impression that it limits the FPS, Example: The wild rift or Call Of Duty even configured to be at 60fps, when connecting the hdmi to Dex cable, it gives the impression that it is locked at 30fps, when disconnecting it returns to 60fps, but this is not just games, is literally all over the rom"

If anyone has HDMI DeX and could produce some logs for us, please do. verbose, error, and dmesg logs would be great.

**Comment by romzip** [comment]
Dex In HDMI mode, sound cannot be transmitted to the monitor. It works normally in common mode
